sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -91,22 +91,77 @@ namespace Azure { namespace Core { namespace Http {
   };
 
   /**
-   * HTTP request method.
-   *
+   * @brief The method to be performed on the resource identified by the Request.
    */
   class HttpMethod final {
   public:
     HttpMethod() = delete;
+
+    /**
+     * @brief Construct a new HTTP Method from a given string.
```

Comment:
...Constructs (see #2299)
```suggestion
     * @brief Constructs `HttpMethod` from string.
```